### DISCUSSION TITLE
ClassIsland改进建议

### DISCUSSION THREAD

**Discussion by huonanwholovecomputer** [ClassIsland/ClassIsland#351] [Q&A]
在时间表配置中新增功能：批量移动

  - 学校的时间安排会根据春、夏、秋、冬进行变化，而更改时间表时只能一个一个编辑，非常的麻烦，这时候，批量选中时间线，向上或向下移动就非常的nice了

**Comment by DryIce-cc** [comment]
$\textcolor{green}{(+)\ 支持：}$已创建#354
